### ISSUE TITLE
Display upgrade prompt window on Help About screen, when available

### ISSUE THREAD

**Issue by simplypeachy** [canton7/SyncTrayzor#217]
Instead of a link to the downloads page, as it's a much nicer way to upgrade.

**Comment by canton7** [comment]
Yeah I should do. It's easier said than done though. With the usual automatic upgrade prompt, we've already downloaded and verified the download (and extracted it, in the case of portable upgrades). Therefore there isn't any UI to show "currently downloading upgrade", etc. We'd also have to handle edge cases, such as the user asking for a manual upgrade while we're in the middle of downloading an update in the background.
